src/libs/ReportUtils.js

```diff
@@ -1108,14 +1145,13 @@ function getIOUReportActionMessage(type, total, participants, comment, currency,
  * @param {Number} amount - IOU amount in cents.
  * @param {String} currency
  * @param {String} comment - User comment for the IOU.
- * @param {Array}  participants - An array with participants details.
  * @param {String} transactionID
  * @param {String} [paymentType] - Only required if the IOUReportAction type is 'pay'. Can be oneOf(elsewhere, payPal, Expensify).
  * @param {String} [iouReportID] - Only required if the IOUReportActions type is oneOf(decline, cancel, pay). Generates a randomID as default.
  * @param {Boolean} [isSettlingUp] - Whether we are settling up an IOU.
  * @returns {Object}
  */
-function buildOptimisticIOUReportAction(type, amount, currency, comment, participants, transactionID, paymentType = '', iouReportID = '', isSettlingUp = false) {
+function buildOptimisticIOUReportAction(type, amount, currency, comment, transactionID, paymentType = '', iouReportID = '', isSettlingUp = false) {
```

Comment:
This change needs reverted.
Right now app crashes on `main` because of this line which was added in https://github.com/Expensify/App/pull/18656
https://github.com/Expensify/App/blob/1b1e562a33c01dbbc2ca81c8994342f95b5aa481/src/libs/ReportUtils.js#L1181

Reply:
Let's discuss a resolution in slack: https://expensify.slack.com/archives/C01GTK53T8Q/p1683719211068359